save-cloud-common/build.gradle.kts

```diff
@@ -81,6 +87,17 @@ kotlin {
                 runtimeOnly(libs.junit.jupiter.engine)
             }
         }
+
+        val linuxX64Main by getting
+        val macosX64Main by getting
+        val mingwX64Main by getting
+
+        val nativeMain by creating {
```

Comment:
why do we need `nativeMain` here?

Reply:
There is an implementation for `linuxX64`, `macosX64` and `mingwX64`

Reply:
discussed offline: gradle has problem with long hierarchy